pkg/workflow/v1/testData/badSyntax.yaml

```diff
@@ -0,0 +1 @@
+key: "value
```

Comment:
This directory should be called `testdata` exactly -- it's a Go convention that prevents their tooling from accidentally picking up test fixtures: https://golang.org/cmd/go/#hdr-Package_lists_and_patterns